app/controllers/v1/sessions_controller.rb

```diff
@@ -26,9 +26,10 @@ def new
 
       StatsD.increment(STATSD_SSO_NEW_KEY,
                        tags: ["context:#{type}", "forceauthn:#{force_authn?}"])
+      type = 'ssoe_slo' if type == 'slo'
       url = url_service.send("#{type}_url")
 
-      if %w[slo ssoe_slo].include?(type)
+      if type == 'ssoe_slo'
         Rails.logger.info("LOGOUT of type #{type}", sso_logging_info)
```

Comment:
its a little confusing that the StatsD above would log a type of 'slo' and then we'd have a Rails logger message of type 'sloe' here.  Could we move the type reassignment on line 29 above the StatsD call?

Reply:
I wound up reworking this anyway because it was complaining about the unsafe send based on user input.